### PR TITLE
Improvement: improve description ui as per feedback

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Improve description UI as per feedback #1710 point 16

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image
image
image
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx [hunk 1]
```diff
@@ -44,13 +45,36 @@ const Description = ({
...
   return (
-    <div className="schema-description tw-flex tw-flex-col tw-h-full tw-min-h-168 tw-relative tw-border tw-border-main tw-rounded-md">
-      <div className="tw-flex tw-items-center tw-px-3 tw-py-1 tw-border-b tw-border-main">
-        <span className="tw-flex-1 tw-leading-8 tw-m-0 tw-text-sm tw-font-medium">
-          Description
-        </span>
+    <div className="schema-description tw-flex tw-flex-col tw-h-full tw-min-h-12 tw-relative">
+      <div className="tw-px-3 tw-py-1 tw-flex">
+        <div>
+          <div data-testid="description tw-overflow-y-auto" id="description">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx:51]:
```suggestion
          <div className="description tw-overflow-y-auto" data-testid="description" id="description">
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx [hunk 2]
```diff
@@ -44,13 +45,36 @@ const Description = ({
...
 }: Props) => {
   return (
-    <div className="schema-description tw-flex tw-flex-col tw-h-full tw-min-h-168 tw-relative tw-border tw-border-main tw-rounded-md">
-      <div className="tw-flex tw-items-center tw-px-3 tw-py-1 tw-border-b tw-border-main">
-        <span className="tw-flex-1 tw-leading-8 tw-m-0 tw-text-sm tw-font-medium">
-          Description
-        </span>
+    <div className="schema-description tw-flex tw-flex-col tw-h-full tw-min-h-12 tw-relative">
+      <div className="tw-px-3 tw-py-1 tw-flex">
+        <div>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/description/Description.tsx:50]:
```suggestion
        <Fragment>
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Suggested a few changes.

File: openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css [hunk 3]
```diff
@@ -156,7 +156,7 @@
   }
 
   .tw-tag {
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css:158]: Use `getCountBadge` instead of this class

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
One comment

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**